src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_checkout.yml

```diff
@@ -68,3 +68,41 @@ winzou_state_machine:
                     do: ["@sylius.state_resolver.order_shipping", "resolve"]
                     args: ["object"]
                     priority: -100
+
+framework:
+    workflows:
+        sylius_order_checkout:
+            type: 'state_machine'
+            marking_store:
+                type: 'method'
+                property: 'checkoutState'
+            supports: 
+                - "Sylius\\Component\\Core\\Model\\Order"
+            initial_marking: cart
+            places:
+                - cart
```

Comment:
How bad idea is to use consts? It will be massively less readable.

![image](https://github.com/Sylius/Sylius/assets/870747/1a51510a-41d6-4406-a4e5-3feaa08af3a0)